tsconfig.json

```diff
@@ -28,12 +24,6 @@
     "noEmit": true,
     "declaration": true
   },
-  "include": [
-    "./src/**/*"
-  ],
-  "exclude": [
-    "/node_modules/",
-    "**/*.test.ts",
-    "**/*.test.tsx"
-  ]
+  "include": ["./src/**/*", "src/vendor/dygraph-c91c859.min.js"],
```

Comment:
I don't see this dygraph js included in the previous version of the tsconfig.json, what is the reason for this being added now?